### ISSUE TITLE
[Suggestion] Stack only once

### ISSUE THREAD

**Issue by Zoiter7** [Nathat23/StackMob-5#197]
Hello, could you add an option for only stack the mob to their limit only once?
I mean after get that, no spawn more mobs for that mob.

**Comment by Nathat23** [comment]
So you want it so that the stack will stack until it reaches max stack size, then will not longer stack, even if stack value is decreased?

**Comment by Zoiter7** [comment]
Yes only stack once for each entity in the configurated radius, but if stack value is decreased increase it, the idea is only have max of 1 stack of mobs no more.
